pkg/resources/syslogng/statefulset.go

```diff
@@ -89,6 +89,7 @@ func syslogNGContainer(spec *v1beta1.SyslogNGSpec) corev1.Container {
 		Args: []string{
 			"--cfgfile=" + configDir + "/" + configKey,
 			"--control=" + socketPath,
+			"--persist-file=" + bufferPath + "/syslog-ng.persist",
```

Comment:
use `filepath.Join` to avoid double slash
```suggestion
			"--persist-file=" + filepath.Join(bufferPath, "/syslog-ng.persist"),
```